corehq/pillows/blobs.py

```diff
@@ -0,0 +1,26 @@
+from couchexport.models import SavedBasicExport
+from pillowtop.listener import BasicPillow
+
+
+class BlobDeletionPillow(BasicPillow):
```

Comment:
I'm trying to convert new pillows to use `ConstructedPillow`(https://github.com/dimagi/pillowtop/blob/master/pillowtop/pillow/interface.py#L112) whenever possible. I don't think there should be any reason we can't do that here. Can you convert it? You can follow the model [used here](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/change_feed/consumer/pillow.py#L39) but with a couch changes feed